bin/test.sh

```diff
@@ -65,13 +65,13 @@ if [ "$SKIP_SETUP" -ne 1 ]; then
     ISTIO_CONTROL=${ISTIO_CONTROL:-istio-control}
 
     kubectl -n bookinfo delete -f samples/bookinfo/platform/kube/bookinfo.yaml --ignore-not-found
-    kubectl -n bookinfo delete -f samples/bookinfo/networking/destination-rule-all-mtls.yaml --ignore-not-found
+    kubectl -n bookinfo delete -f samples/bookinfo/networking/destination-rule-all.yaml --ignore-not-found
```

Comment:
In the makefile I am adding both DR and Policy to the simple test - I think we should recommend as best practice for all users to be explicit and include the desired settings for each namespace instead of assuming or relying on install options. 

It may be better to run bookinfo with mtls on - we do test mtls off with simple.

Reply:
We just saw that the pipeline was broken after #42 - so we did the simplest fix after finding the problem. Currently, the pipeline "only" installs the default given by the [global|value].yaml.